applications/IgaApplication/iga_application.h

```diff
@@ -76,7 +76,7 @@ class KratosIgaApplication
     ///@name Operations
     ///@{
 
-    virtual void Register();
+    void Register() override;
```

Comment:
[Also in the Application-Template?](https://github.com/KratosMultiphysics/Kratos/blob/a741949882c360ae3f738839a793b6c1a449f5a7/kratos/templates/application#L98)

Reply:
👍 Maybe also the Info-stuff at the end? (actually I am just not sure... if it is ok then I can also do the job)